src/Tribe/Views/V2/View.php

```diff
@@ -1568,4 +1569,44 @@ protected function filter_display_events_bar( $display ) {
 
 		return $display;
 	}
+
+	/**
+	 * Returns the iCal data we're sending to the view.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
```

Comment:
you're returning an object in the function, but in the docs you're returning an array.